Assets/Scripts/Command/ChangeEntityName.cs

```diff
@@ -13,11 +11,15 @@ public class ChangeEntityName : Command
     private Guid entityId;
     private string newName, oldName;
 
-    public ChangeEntityName(Guid entityId, string newName, string oldName)
+    // Dependencies
+    private EditorEvents _editorEvents;
```

Comment:
Commands sollten nie Referenzen auf irgendwelche Objekte speichern. Die Idee ist, dass man später commands serialisieren und deserialisierten können soll. Stattdessen würde ich die `editorEvents` mit in der `Do` und `Undo` Funktion übergeben

Reply:
Könntest du mal ausprobieren, ob du aus der Basisklasse Command an die EditorEvents Klasse kommst? Und dazu vielleicht noch die Frage, ob wir dann ggf. zwei Event Klassen definieren sollten: eine für Events, die den Editor betreffen und eine, welche die Szene betreffen?

Reply:
Stimmt, das geht aktuell nicht. Mir fallen gerade zwei Lösungen dafür ein.
- Die `EditorEvents` Klasse in ein eigenes Assembly stecken. Dieses Assembly könnte dann auf `SceneState` referenzieren, um an `Command` zu kommen
- `Command` und `CommandHistoryState` werden in die `EditorStates` Assembly verschoben. 

Ansonsten ist noch zu überlegen, ob wir die Unterscheidung zwischen `SceneStates` und `EditorStates` überhaupt brauchen

Die Events auf Editor und Scene events aufzuteilen ist auch eine gute Idee